### PR TITLE
test to container image tag entity, Airgun

### PR DESCRIPTION
**Pull request by tstrych**
@oshtaier please review,
test results:
```
============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 2 items2018-10-03 15:50:14 - conftest - DEBUG - Collected 2 test cases


test_containerimagetag.py                                              [100%]

============================== 0 tests deselected ==============================
========================== 2 passed in 271.89 seconds ==========================
```

### CODE REVIEW

**Comment by tstrych** [comment]
test results:
```
============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 2 items2018-10-03 17:21:13 - conftest - DEBUG - Collected 2 test cases


test_containerimagetag.py                                              [100%]

============================== 0 tests deselected ==============================
========================== 2 passed in 267.92 seconds ==========================
```

File: tests/foreman/ui_airgun/test_containerimagetag.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.decorators import fixture, tier2
+
+
+@fixture(scope="module")
+def org():
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_containerimagetag.py]: `def module_org`

File: tests/foreman/ui_airgun/test_containerimagetag.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        docker_upstream_name='busybox',
+        product=product,
+        url=u'https://registry-1.docker.io',
+    ).create()
+    repo.sync()
+    return repo
+
+
+@tier2
+def test_positive_search(session, org, product, sync_repo):
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_containerimagetag.py]: definitely join these two test cases into one

File: tests/foreman/ui_airgun/test_containerimagetag.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+"""WebUI tests for the Docker Image Tags feature.
+
+:Requirement: Docker
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_containerimagetag.py]: :Requirement: Docker Image Tag

File: tests/foreman/ui_airgun/test_containerimagetag.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        assert sync_repo.name == search[0]['Repository Name']
+
+
+def test_positve_read_details(session, org, product, sync_repo):
+    with session:
+        session.organization.select(org_name=org.name)
+        values = session.containerimagetag.read('latest')
+        assert product.name == values['details']['product']
+        assert sync_repo.name == values['details']['repository']
+        assert values['lce']['table'][0]['Environment'] == 'Library'
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_containerimagetag.py]:
```python
from robottelo.constants import ENVIRONMENT
...
assert values['lce']['table'][0]['Environment'] == ENVIRONMENT
```

File: tests/foreman/ui_airgun/test_containerimagetag.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    return entities.Organization().create()
+
+
+@fixture(scope="module")
+def product(org):
+    return entities.Product(organization=org).create()
+
+
+@fixture(scope="module")
+def sync_repo(product):
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_containerimagetag.py]: module_repository

File: tests/foreman/ui_airgun/test_containerimagetag.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+from robottelo.decorators import fixture, tier2
+
+
+@fixture(scope="module")
+def org():
+    return entities.Organization().create()
+
+
+@fixture(scope="module")
+def product(org):
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_containerimagetag.py]:
```python
def module_product(module_org)
```

**Comment by tstrych** [comment]
test results:
```
========================================== test session starts ==========================================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 1 item                                                                                        
2018-10-04 11:05:26 - conftest - DEBUG - Collected 1 test cases


tests/foreman/ui_airgun/test_containerimagetag.py .                                               [100%]

========================================== 0 tests deselected ===========================================
====================================== 1 passed in 189.42 seconds =======================================
```

**Pull request merged**